ui/components/core/src/components/Histogram.svelte

```diff
@@ -15,105 +15,81 @@
    */
 
   // Imports
-  import { VegaLite, type VisualizationSpec } from "svelte-vega";
+  import { Bar } from "svelte-chartjs";
+  import {
+    Chart,
+    Title,
+    Tooltip,
+    Legend,
+    BarElement,
+    CategoryScale,
+    LinearScale,
+    type ChartDataset,
+  } from "chart.js";
+  import zoomPlugin from "chartjs-plugin-zoom";
+  import type { DatasetStat } from "..";
 
-  import type { DatasetStat } from "../lib/types/datasetTypes";
-
-  // Exports
+  // Define props
   export let hist: DatasetStat;
-  export let maxHeight = 48;
-  export let hideTitle = false;
 
-  // Calculate histogram height
-  let h = 7.5 * hist.histogram.length;
-  h = h < 125 ? 125 : h;
+  // Color palette for stacked charts
+  const colors = ["#771E5F", "#1E7736", "#1E5F77", "#77361E"];
 
-  // Vega-Lite spec
-  const spec: VisualizationSpec = {
-    $schema: "https://vega.github.io/schema/vega-lite/v5.json",
-    config: {
-      style: {
-        cell: {
-          stroke: "transparent",
-        },
-      },
-    },
-    params: [
-      {
-        name: "highlight",
-        select: {
-          type: "point",
-          on: "mouseover",
-        },
+  // Separate splits
+  let splits: string[] = [...new Set(hist.histogram.map((item) => String(item.split)))];
+  let datasets: ChartDataset[] = [];
```

Comment:
for consistency, I think its easier to read if variable are named after / consistent with their type
```
let chartDatasets: ChartDataset[] = []
```